src/commands/run.rs

```diff
@@ -58,15 +59,27 @@ const CONFIGS_HOST_IMPLEMENTORS: [&str; 3] =
 #[cfg(feature = "sql")]
 const SQL_HOST_IMPLEMENTORS: [&str; 1] = ["sql.postgres"];
 
-pub async fn handle_run(module: impl AsRef<Path>, toml_file_path: impl AsRef<Path>) -> Result<()> {
+pub type IORedirects = slight_runtime::IORedirects;
+
+#[derive(Clone, Default)]
+pub struct RunArgs {
+    pub module: PathBuf,
+    pub slightfile: PathBuf,
```

Comment:
I think slightfile has already been included as a global argument to `slight` CLI.

Reply:
We have the `slightfile` being pased via the `toml_file_path` arg currently~ I think David is just grouping it all together w/ `RunArgs`.

Reply:
Yes, that is correct. Rather than continuing to expand the function signature, we can add fields to the `RunArgs` struct as needed.

Reply:
Okay, slightfile arg is being used in `slight secret` command as well. Just want to make sure that it will continue to work.